src/DIRAC/FrameworkSystem/Service/PlottingHandler.py

```diff
@@ -5,7 +5,6 @@
 import hashlib
```

Comment:
I think this whole service could be removed, together with PlottingClient client.